### ISSUE TITLE
[Announcement] shaka-player-users list change

### ISSUE THREAD

**Issue by joeyparrish** [google/shaka-player#1095]
TL;DR: [shaka-player-users](https://groups.google.com/forum/#!forum/shaka-player-users) is now for announcements only, please post questions/issues [on github](https://github.com/google/shaka-player/issues/new)

<hr>

When the [shaka-player-users](https://groups.google.com/forum/#!forum/shaka-player-users) list was first set up, is was intended to be a place for announcements, questions, and general discussion.  In practice, though, the community seems to prefer to ask questions and start discussions on github.  We have also discovered that it is easier to keep up with incoming questions via github than via the mailing list.

For these reasons, we have decided to **repurpose this mailing list for announcements only**.  On Wednesday, November 1, we will change the settings on the list so that only Shaka Player team members will be able to post.  Questions and discussions can be posted to [our github issues page](https://github.com/google/shaka-player/issues/new).  Our documentation will be updated to reflect this change.

In future, the list will be used for [announcements about new releases](https://groups.google.com/forum/#!topic/shaka-player-users/5imVcIRk8vI) (historically 1-4 per month) and surveys about features and support (planning for 1-4 per year).

Thanks!
